Framework/src/CheckerDataProcessor.cxx

```diff
@@ -31,6 +31,7 @@ using namespace repository;
 namespace checker {
 
 // TODO do we need a CheckFactory ? here it is embedded in the Checker
+// TODO maybe we could use the CheckerDataProcessorFactory
```

Comment:
I am not sure if I understand - `CheckerDataProcessorFactory` for both checkers and checks?

Reply:
yes, with a rename :)